src/ProjectOrigin.VerifiableEventStore/Services/TransactionStatusCache/TransactionStatusService.cs

```diff
@@ -31,14 +31,14 @@ public async Task<TransactionStatusRecord> GetTransactionStatus(string transacti
         }
         else
         {
-            var a = _eventStore.GetTransactionStatus(transactionId);
-            return new TransactionStatusRecord(TransactionStatus.Unknown);
+            var status = await _eventStore.GetTransactionStatus(transactionId);
+            return new TransactionStatusRecord(status);
         }
     }
 
     public Task SetTransactionStatus(string transactionId, TransactionStatusRecord record)
     {
-        _logger.LogInformation($"Setting transaction status for {transactionId} to {record.NewStatus}");
+        _logger.LogDebug($"Setting transaction status for {transactionId} to {record.NewStatus}");
```

Comment:
During investigation of https://github.com/project-origin/registry/issues/122 I had great use of this log statement. I can see that it makes sense to use "Debug" here, but maybe it would be nice, if changing the log level on the Helm chart was easy using the values-file.

Reply:
Maybe LogTrace? then the service can be configured to trace level, if you want it to output these sort of messages?
I would probably find it a bit chatty for a normal log.

Reply:
agreed on logTrace